index.html

```diff
@@ -10,7 +10,54 @@
     <link rel="stylesheet" href="styles/index.css" />
 </head>
 <body>
-    <script src="./src/index.js"></script>
+    <header class="header">
+        <span>
+            <h1>Weather Report</h1>
+            <h2>For the city of
+                <h3 id="cityname">Seattle</h3>
+            </h2>
+        </span>
+    </header>
+
+    <section class="temperature_section">
+        <h2>Temperature</h2>
+        <div id="tempsection">
+            <div id="changeTemp">
+                <span id="upTempControl">⬆️</span>
+                <span id="temp">69</span>
+                <span id="downTempControl">⬇️</span>
+            </div>
+            <button id="realTimeTemp">Get Realtime Temperature</button>
+        </div>
+    </section>
+
+    <section class="garden_section">
+        <h2>Garden</h2>
+        <div id="whole_garden">
+            <div id="sky">☁️☁️☁️☁️☁️☁️☁️☁️☁️☁️☁️</div>
+            <div id="landscape">🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷</div>
```

Comment:
```suggestion
            <p id="sky">☁️☁️☁️☁️☁️☁️☁️☁️☁️☁️☁️</p>
            <p id="landscape">🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷</p>
```
Recommending `<p>` tag for semantic meaning